Rakefile

```diff
@@ -13,10 +13,3 @@ Yast::Tasks.configuration do |conf|
   conf.install_locations["library/desktop/groups/*.desktop"] = File.join(Packaging::Configuration::DESTDIR, "/usr/share/applications/YaST2/groups/")
   conf.install_locations["library/general/hooks/README.md"] = File.join(Packaging::Configuration::DESTDIR, "/var/lib/YaST2/hooks/")
 end
-
-# define additional creation of legacy symlinks during installation
-task :install do
-  sh "/usr/bin/mkdir -p #{Packaging::Configuration::DESTDIR}/sbin"
```

Comment:
can we detect that usrmerge variable also during runtime? As this install task is also used locally when testing ( not sure how much developers use this ).

Reply:
well, you could check whether /sbin is a symlink and skip the block then.

Reply:
I do not know where it might be used, but we should not rely on old symlinks anyway...